client/src/components/Explorer/ExplorerPanel.js

```diff
@@ -137,19 +137,21 @@ class ExplorerPanel extends React.Component {
 
     return (
       <FocusTrap
-        tag="nav"
+        tag="div"
+        role="dialog"
         className="explorer"
         paused={paused || !page || page.isFetching}
         focusTrapOptions={{
-          initialFocus: '.c-explorer__close',
+          initialFocus: '.c-explorer__header',
           onDeactivate: onClose,
         }}
       >
-        <Button className="c-explorer__close u-hidden" onClick={onClose}>
+        <Button className="c-explorer__close visuallyhidden" onClick={onClose}>
           {STRINGS.CLOSE_EXPLORER}
         </Button>
         <Transition name={transition} className="c-explorer">
-          <div key={path.length} className="c-transition-group">
+          <nav key={path.length} className="c-transition-group" aria-labelledby="nav-page-explorer-header">
+            <h2 className="visuallyhidden" id="nav-page-explorer-header">Page Explorer</h2>
```

Comment:
There are a few issues here:

- Should be "Page explorer", we don’t tend to use Title Case in Wagtail.
- The label should be translatable via our STRINGS config.
- This is inside a transition group – so in-between transitions the page would contain two copies of everything inside `<Transition>`, and we don’t want to have two headings with the same label, or duplicate ids.

I don’t think the "duplicate headings and id" would be likely to cause problems but they feel worth avoiding nonetheless. I’d suggest to change the `<Transition>` instead to:
```
<Transition name={transition} className="c-explorer" component="nav" label={STRINGS.PAGE_EXPLORER}>
```

This component already supports setting a `component`, it’ll just need to be updated to also support a `label` prop forwarding to `aria-label` on `CSSTransitionGroup`.

Reply:
Updated in https://github.com/wagtail/wagtail/pull/5394/commits/03d9ad9989eca64b339ac8747ff7433e42286e0d